### PR TITLE
Make mail_new_activity no-op silently

### PR DESCRIPTION
**Pull request by arandomandy**

### CODE REVIEW

**Comment by arandomandy** [comment]
> I think this is related to #731
> 
> I would suggest adding a test for this fix at https://github.com/lobsters/lobsters/blob/master/spec/scripts/mail_new_activity_spec.rb so that in the future this issue doesn't reappear and so that we can show that the fix does what it says on the tin.

**Comment by arandomandy** [comment]
> I think this is related to #731
> 
> I would suggest adding a test for this fix at https://github.com/lobsters/lobsters/blob/master/spec/scripts/mail_new_activity_spec.rb so that in the future this issue doesn't reappear and so that we can show that the fix does what it says on the tin.

I thought about this, but I couldn't figure out a way to test it since I am so new to RoR. If anyone knows how to go about testing it, feel free to jump in.

cc @thomasdziedzic @pushcx @hmadison

File: script/mail_new_activity.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def story_subject(story, prefix = "")
 
   mailing_list_users = User.where("mailing_list_mode > 0").select(&:is_active?)
 
-  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last.id).to_i
+  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last && Story.last.id).to_i
```
Comment (thomasdziedzic) [inline script/mail_new_activity.rb]: I think you can use the safe navigation operator here http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

```suggestion
  last_story_id = (Keystore.value_for(LAST_STORY_KEY) || Story.last&.id).to_i
```

File: script/mail_new_activity.rb [hunk 2]
```diff
@@ -142,8 +142,7 @@ def story_subject(story, prefix = "")
 
   # repeat for comments
 
-  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) ||
-    Comment.last.id).to_i
+  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) || Comment.last && Comment.last.id).to_i
```
Comment (thomasdziedzic) [inline script/mail_new_activity.rb]: Same comment here with the safe navigation operator.

```suggestion
  last_comment_id = (Keystore.value_for(LAST_COMMENT_KEY) || Comment.last&.id).to_i
```

File: script/mail_new_activity.rb (hunk 1 above)
Comment (pushcx) [reply script/mail_new_activity.rb]: We have a custom rubocop linter preventing it.

**Comment by pushcx** [comment]
I'm OK without a test - someone can only run into this bug once when first starting a site, and it's harmless.

Thanks for fixing this, @arandomandy and thanks for the review, @thomasdziedzic.

**Pull request merged**